packages/liveblocks-core/src/room.ts

```diff
@@ -2082,6 +2087,11 @@ export function createRoom<
     flushNowOrSoon();
   }
 
+  function clear() {
+    context.undoStack.length = 0;
```

Comment:
does it matter that this is mutable, vs. setting to `[]`?

Reply:
Both should work. In this case, I used this mutable version because of the inherent mutable nature of the stack — so it's similarly mutating a single array instance, just like `.pop()` and `.push()` and our other `.length` mutation is doing. To make this extra clear, I've made these `readonly` properties on `context` now in f81715c0ebc8be23d0ca81a1b79f030baaad4047. This ensures we can never accidentally store a reference to an "old stack" somewhere.